core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpConnectionContext.java

```diff
@@ -327,6 +295,32 @@ void addTableUpdateDetails(Utf8String tableNameUtf8, TableUpdateDetails tableUpd
         tableUpdateDetailsUtf8.put(tableNameUtf8, tableUpdateDetails);
     }
 
+    @Override
+    protected void doInit() {
+        if (recvBuffer.getBufStart() == 0) {
+            recvBuffer.of(configuration.getRecvBufferSize(), configuration.getMaxRecvBufferSize());
+            goodMeasurement = true;
+        }
+
+        authenticator.init(socket, recvBuffer.getBufStart(), recvBuffer.getBufEnd(), 0, 0);
+        if (authenticator.isAuthenticated() && securityContext == DenyAllSecurityContext.INSTANCE) {
```

Comment:
this code makes no sense, if this is ever DenyAllSecurityContext authorizeLineTcp() will throw exception in the accept loop